src/components/TopNav/NavMenu.js

```diff
@@ -40,7 +43,10 @@ export const mapStateToProps = (state, ownProps) => {
 
   return {
     ...ownProps,
-    ticketIds: Object.keys(state.tickets.map).filter(idContainsANumberAndIsNotCurrent)
+    ticketIds: Object.entries(state.tickets.map)
```

Comment:
So, this means we display tickets that have a non-null second position in the array? Where is this array from?

Reply:
The filter statement is operating on the key-value pairs returned by Object.entries; the second position in the key-value pair is the ticket object, which is updated to null when the menu button for removing a ticket is clicked. This is a workaround because redux-persist does not track removal of keys, only updates to new non-undefined values.